test/e2e/framework/framework.go

```diff
@@ -169,23 +187,27 @@ func (f *Framework) BeforeEach() {
 		Expect(err).NotTo(HaveOccurred())
 	}
 
-	By("Building a namespace api object")
-	namespace, err := f.CreateNamespace(f.BaseName, map[string]string{
-		"e2e-framework": f.BaseName,
-	})
-	Expect(err).NotTo(HaveOccurred())
+	if !f.SkipInitCloudProvider {
+		cloud, err := cloudprovider.InitCloudProvider(oci.ProviderName(), cloudConfigFile)
+		Expect(err).NotTo(HaveOccurred())
+		f.CloudProvider = cloud
+	}
 
-	f.Namespace = namespace
+	if !f.SkipNamespaceCreation {
```

Comment:
will these ever be false if you're setting htem to true above after creating the framework?

Reply:
It's false in the Service tests